solver-functions/solver/unified_builder.py

```
@@ -0,0 +1,679 @@
+
+
+def _has_night_shift(requirements: ShiftRequirementDict) -> bool:
+    """要件キーに「夜勤」が含まれるか → 夜勤施設判定"""
+    return any("夜勤" in key for key in requirements["requirements"])
+
+
+def _non_operational_days(
+    requirements: ShiftRequirementDict, days_in_month: int
+) -> set[int]:
+    """要件エントリがない日 → 非稼働日"""
+    target_month = requirements["targetMonth"]
```

Ruff (0.15.0)

[error] Local variable `target_month` is assigned to but never used

Remove assignment to unused variable `target_month`

(F841)